app/models.js

```diff
@@ -212,3 +213,22 @@ class Pod extends K8sModel {
 }
 
 exports.Pod = Pod;
+
+
+class ToolDeployment {
+
```

Comment:
No blank line here - did you run `npm lint`?

Reply:
😇 ...I forgot to run `npm lint`